src/encoded/tests/test_batch_download.py

```diff
@@ -44,7 +44,7 @@ def lookup_column_value_validate():
         'award.project': 'Roadmap',
         '@id': '/experiments/ENCSR751ISO/',
         'level.name': '',
-        '@type': ['Experiment', 'Dataset', 'Item']
+        '@type': 'Experiment,Dataset,Item'
```

Comment:
that looks totally bizarre to me.

Reply:
Yeah. Actually, I wrote this fixture. I think when I ran it, I dint validate it for a list. So, it passed locally. Then later I added list to the validation data and dint mind running it.